packages/aws-cdk/jest.config.js

```diff
@@ -7,12 +7,18 @@ module.exports = {
             // DO NOT LOWER THESE VALUES!
             // If you need to break glass, open an issue to re-up the values with additional test coverage
             statements: 81,
-            branches: 68,
+            branches: 66,
```

Comment:
Pretty rude 😂

Reply:
Lol, and this is was even after writing more tests.